packages/perseus/src/widgets/__tests__/grapher.cypress.jsx

```diff
@@ -38,15 +38,19 @@ describe("Grapher widget", () => {
             );
 
             // Act
-            cy.get(GRAPHIE).then((node) => {
-                const {left, top} = node[0].getBoundingClientRect();
-                cy.get(POINTS)
-                    .eq(0)
-                    .dragTo({x: left + 360, y: top + 180});
-                cy.get(POINTS)
-                    .eq(1)
-                    .dragTo({x: left + 340, y: top + 140});
-            });
+            cy.get(GRAPHIE)
+                .should("exist")
```

Comment:
Does this basically assert that the previous `.get()` returned an element?  If so, I love it!